tributors/utils/prompt.py

```diff
@@ -27,3 +27,32 @@ def request_input():
             lines.append(line)
 
     return "\n".join(lines)
+
+
+def choice_prompt(prompt, choices, choice_prefix=None, multiple=False):
+    """Ask the user for a prompt, and only return when one of the requested
+       options is provided.
+       Parameters
+       ==========
+       prompt: the prompt to ask the user
+       choices: a list of choices that are valid.   
+       multiple: allow multiple responses (separated by spaces)
+    """
+    choice = None
+    print(prompt)
+    get_input = getattr(__builtins__, "raw_input", input)
```

Comment:
for myself -- is that for py2/3 compatibility? forgot already why it might be needed and there is no comment to enlighten silly ones like me

Reply:
Yes it used to be raw_input in older python - I would really really hope nobody needs this anymore.